From 0327c427d62f671eced067c6d9b69f4e216a8cac Mon Sep 17 00:00:00 2001
From: Adam Jackson <ajax@redhat.com>
Date: Mon, 17 Jun 2019 13:36:08 -0400
Subject: [PATCH] makekeys: Detach ourselves from X headers entirely

Subsequent to a121b7b0c210efe10bf93453b29050282324c906 we are no longer
building makekeys with enough -I/foo/bar to find the X11 headers, so if
they're not in a system include path, things fail. Since this utility is
only needed at build time, there's no real reason to demand the X
headers be installed for both the build and target machines if cross-
compiling, we can just assume a vaguely ANSI environment instead.

Tested-by: Niclas Zeising <zeising@daemonic.se>
Reviewed-by: Keith Packard <keithp@keithp.com>
Reviewed-by: Matt Turner <mattst88@gmail.com>
Signed-off-by: Adam Duskett <aduskett@gmail.com>
---
 src/util/makekeys.c | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/src/util/makekeys.c b/src/util/makekeys.c
index bcb5b7d5..07563315 100644
--- a/src/util/makekeys.c
+++ b/src/util/makekeys.c
@@ -35,8 +35,10 @@ from The Open Group.
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
+#include <stdint.h>
+#include <inttypes.h>
 
-#include "../Xresinternal.h"
+typedef uint32_t Signature;
 
 #define KTNUM 4000
 
-- 
2.21.0

